app/src/main/java/com/mpieterse/stride/ui/layout/central/components/CreateNotificationDialog.kt

```
@@ -218,3 +415,288 @@ fun CreateNotificationDialog(
+                    contentColor = Color.Black
+                ),
+                shape = RoundedCornerShape(8.dp)
+            ) {
+                Text("Cancel")
+            }
+        }
+    )
+}
+
+@Composable
+private fun ScrollableNumberPicker(
```

detekt (1.23.8)

[warning] The function ScrollableNumberPicker appears to be too complex based on Cognitive Complexity (complexity: 39). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)